packages/core/ui/core/view/index.ios.ts

```diff
@@ -54,9 +54,21 @@ export class View extends ViewCommon implements ViewDefinition {
 		return (this._privateFlags & PFLAG_FORCE_LAYOUT) === PFLAG_FORCE_LAYOUT;
 	}
 
+	requestlayoutIfNeeded() {
```

Comment:
Maintain same camel case throughout:
```suggestion
	requestLayoutIfNeeded() {
```